tests/Aspire.Hosting.Tests/Directory.Build.props

```diff
@@ -1,8 +1,7 @@
 <Project>
+  <PropertyGroup>
+    <IsAspireHost>true</IsAspireHost>
```

Comment:
Is there a way to make it so this stays defined in the .csproj?

Reply:
We can keep it there *also*! It is needed basically for this in `tests/Shared/RepoTesting/Aspire.Testing.Repo.props` which gets imported early - before the project file itself:
```xml
  <!-- IsAspireHost property might not be available here, so allow infering that from the project name -->
  <ImportGroup Condition="'$(RepoRoot)' != 'null' and '$(TestsRunningOutsideOfRepo)' != 'true' and ($(MSBuildProjectName.EndsWith('AppHost')) or '$(IsAspireHost)' == 'true')">
    <Import Project="$(RepoRoot)src/Aspire.Hosting.AppHost/build/Aspire.Hosting.AppHost.props" Condition="Exists('$(RepoRoot)src/Aspire.Hosting.AppHost/build/Aspire.Hosting.AppHost.props')" />
  </ImportGroup>
```

We are using `($(MSBuildProjectName.EndsWith('AppHost')) or '$(IsAspireHost)' == 'true')` to determine whether it is an aspire host or not. And in this case the name does not help. Open to any suggestions of course.